docs/UsersGuide/source/SRWAppOverview.rst

```diff
@@ -146,7 +146,7 @@ executables listed in :numref:`Table %s <exec_description>` will be located in t
    +------------------------+---------------------------------------------------------------------------------+
    | orog                   | Generates orography, land mask, and gravity wave drag files from fixed files    |
    +------------------------+---------------------------------------------------------------------------------+
-   | regional_esg_grid      | Generates an ESG regional grid based on a user-defined namelist                |
+   | regional_esg_grid      | Generates an ESG regional grid based on a user-defined namelist                 |
```

Comment:
I didn't understand why that table didn't show in the html version, we need to make sure it does this time.